File: Web Application/frontend/src/files/public/uploads/palindrome.r
palindrome <- function(n){
rev = 0
    num = n

    while (n > 0) {
      r = n %% 10
      rev = rev * 10 + r
      n = n %/% 10
    }

    if (rev == num)
    {
      print(paste("Number is palindrome :", rev))
    }
    else{
      print(paste("Number is not palindrome :", rev))
    }
}
palindrome(121)

# exec("""\npalindrome <- function(n){\nrev = 0\n    num = n\n\n    while (n > 0) {\n      r = n %% 10\n      rev = rev * 10 + r\n      n = n %/% 10\n    }\n\n    if (rev == num)\n    {\n      print(paste("Number is palindrome :", rev))\n    }\n    else{\n      print(paste("Number is not palindrome :", rev))\n    }\n}\npalindrome(121)\n""")

File: Web Application/frontend/src/files/public/uploads/palindrome.py
def palindrome(num):
    temp = num
    rev = 0
    while(num > 0):
        dig = num % 10
        rev = rev * 10 + dig
        num = num // 10
    if(temp == rev):
        print("Number is palindrome")
    else: 
        print("Number is not palidrome")

palindrome(131)

# exec("""\ndef palindrome(num):\n    temp = num\n    rev = 0\n    while(num > 0):\n        dig = num % 10\n        rev = rev * 10 + dig\n        num = num // 10\n    if(temp == rev):\n        print("Number is palindrome")\n    else: \n        print("Number is not palidrome")\n\npalindrome(131)\n""")